tasks/release.rake

```diff
@@ -3,7 +3,7 @@
 namespace :release do
   desc 'Create a new prerelease, optionally publishing to rubygems.org'
   task :pre, :publish do |_t, args|
-    cmd = 'gem bump --push --version pre'
+    cmd = 'gem bump --push --version pre'.dup
     cmd.concat(' --tag --release') if args.publish == 'true'
```

Comment:
Why was it necessary to add the `.dup`?

Reply:
Without the dup, it threw an error that "you cannot modify a frozen string".  For some reason (and it may be more recent versions of Ruby?), the literal string was apparently considered frozen, and that causes the `concat` to fail now.

In my searching online, I found a stack exchange question about this, and the highest-rated answer was to use "dup" which creates a duplicate of it.

I can keep digging to try and find out WHY it's treating that string as frozen, though.

Reply:
Hmm, strange.  I can't duplicate this behavior in IRB (using simple string concats), so there may be something going on that I'm not seeing.

Reply:
Ah, I think I figured it out.  At the top of the file is `#frozen_string_true", which apparently will cause all string literals to become immutable.  The "dup" basically takes a copy of the immutable string and makes it a "regular" (ie, mutable) string.

Not sure why we weren't seeing this behavior before (though it only shows up now if you use the release:pre[true] task)

Reply:
Ah, the "frozen string" setting -- makes sense.